modules/internal/ArrayViewSlice.chpl

```diff
@@ -90,7 +90,8 @@ module ArrayViewSlice {
     }
 
     iter these(param tag: iterKind) ref
-      where tag == iterKind.standalone && !localeModelHasSublocales {
+      where tag == iterKind.standalone && !localeModelHasSublocales &&
+           __primitive("method call resolves", privDom, "these", tag) {
```

Comment:
I could live with this for now, but how do we want users to write these kinds of iterators in the future?